client/package.json

```diff
@@ -11,7 +11,9 @@
   "dependencies": {
     "core-js": "^3.6.4",
     "mdbvue": "6.7.1",
-    "vue": "^2.6.11"
+    "pdfjs-dist": "^2.3.200",
+    "vue": "^2.6.11",
+    "vue-loading-overlay": "^3.3.2"
```

Comment:
From my fast research you can use https://vuetifyjs.com/en/components/progress-circular/ instead.
I don't think we need another library, but you will decide, maybe there is a reason ;)

Reply:
Ok, this version is without vuetify xD

Reply:
I'm gonna look into that when router is merged into master and i can get back to work on this. Thanks for suggestion

Reply:
Progress circular is just a image of loading, while `vue-loading-overlay` is well overlay. It makes background little blurred and progress circle is on full screen. I would stick with my solution because writing the whole overlay by myself looks kinda unnecesarry